src/iotjs_module_tcp.cpp

```diff
@@ -282,7 +273,8 @@ void AfterWrite(uv_write_t* req, int status) {
   args.Add(JVal::Number(status));
 
   // Make callback.
-  MakeCallback(jcallback, JObject::Null(), args);
+  JObject undefined = JObject::Undefined();
```

Comment:
I think if JObject::Undefined() would be static again, we could remove a lot of these local declarations.